src/panels/config/devices/device-detail/ha-device-card-mqtt.ts

```diff
@@ -0,0 +1,51 @@
+import { DeviceRegistryEntry } from "../../../../data/device_registry";
+import { removeMQTTDeviceEntry } from "../../../../data/mqtt";
+import {
+  LitElement,
+  html,
+  customElement,
+  property,
+  TemplateResult,
+  CSSResult,
+  css,
+} from "lit-element";
+import { showConfirmationDialog } from "../../../../dialogs/generic/show-dialog-box";
+import { HomeAssistant } from "../../../../types";
+
+@customElement("ha-device-card-mqtt")
+export class HaDeviceCardMqtt extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public device!: DeviceRegistryEntry;
+
+  protected render(): TemplateResult {
+    return html`
+      <mwc-button class="warning" @click="${this._confirmDeleteEntry}">
+        ${this.hass.localize("ui.panel.config.devices.delete")}
+      </mwc-button>
+    `;
+  }
+
+  protected firstUpdated(changedProps) {
+    super.firstUpdated(changedProps);
+  }
+
+  private async _deleteEntry(): Promise<void> {
+    await removeMQTTDeviceEntry(this.hass!, this.device.id);
+  }
+
+  private _confirmDeleteEntry(): void {
+    showConfirmationDialog(this, {
+      text: this.hass.localize("ui.panel.config.devices.confirm_delete"),
+      confirm: () => this._deleteEntry(),
+    });
+  }
+
+  static get styles(): CSSResult {
+    return css`
+      mwc-button.warning {
+        margin-right: auto;
```

Comment:
That seems weird to add here. Why is that ? 

Also the coloring is already available in `haStyle` or something, I think ?

Reply:
I stole it from here:
https://github.com/home-assistant/frontend/blob/5bb60e899ce80b5f7dca5b06dbaff7986f930917/src/panels/config/entities/entity-registry-settings.ts#L242-L245

Do you mean to use this instead:
https://github.com/home-assistant/frontend/blob/5bb60e899ce80b5f7dca5b06dbaff7986f930917/src/resources/ha-style.ts#L36

Reply:
These: https://github.com/home-assistant/frontend/blob/dev/src/resources/styles.ts#L200-L202

Reply:
The `margin-right: auto` is to pull the secondary button to the left in a dialog normally

Reply:
Thanks! Fixed now.